tests/mockserver.py

```diff
@@ -222,6 +224,14 @@ def ssl_context_factory(keyfile='keys/localhost.key', certfile='keys/localhost.c
          )
 
 
+def broken_ssl_context_factory(keyfile='keys/localhost.key', certfile='keys/localhost.crt', cipher_string='DEFAULT'):
```

Comment:
Maybe it should be renamed.

Reply:
Could you please explain what this code does? Does it 1) add cipher_string support, and 2) disables default ciphers which come from TLSv1_2? And the purpose is to create servers which use outdated ciphers not supported by current http clients by default?

If so, what do you think about 

1) removing this function,
2) adding cipher_string argument to ssl_context_factory, None by default,
3) when cipher_string is not None, ssl_context_factory becomes broken_ssl_context_factory,
4) add a comment which explains why `ctx.set_options(SSL.OP_CIPHER_SERVER_PREFERENCE | SSL.OP_NO_TLSv1_2)` line exists.